tests/integration/test_pagination.py

```diff
@@ -0,0 +1,13 @@
+from copy import copy
+
+
+def test_get_one_and_many_dataset_order(client):
```

Comment:
Nice work adding coverage for pagination!

Reply:
I felt that part of the test_dataset was really testing pagination so I moved that here.